Delivered-To: raj.khem@gmail.com
Received: by 10.90.86.4 with SMTP id j4cs313305agb;
        Sat, 8 Jan 2011 16:45:19 -0800 (PST)
Received: by 10.216.153.210 with SMTP id f60mr573848wek.114.1294533918335;
        Sat, 08 Jan 2011 16:45:18 -0800 (PST)
Return-Path: <yann.morin.1998@anciens.enib.fr>
Received: from smtp.smtpout.orange.fr (smtp01.smtpout.orange.fr [80.12.242.123])
        by mx.google.com with ESMTP id n4si33737071wej.152.2011.01.08.16.45.17;
        Sat, 08 Jan 2011 16:45:18 -0800 (PST)
Received-SPF: neutral (google.com: 80.12.242.123 is neither permitted nor denied by best guess record for domain of yann.morin.1998@anciens.enib.fr) client-ip=80.12.242.123;
Authentication-Results: mx.google.com; spf=neutral (google.com: 80.12.242.123 is neither permitted nor denied by best guess record for domain of yann.morin.1998@anciens.enib.fr) smtp.mail=yann.morin.1998@anciens.enib.fr
Received: from roazhon.bzh.lan ([90.32.245.227])
	by mwinf5d24 with ME
	id tClC1f0024v5z3u03ClGDX; Sun, 09 Jan 2011 01:45:17 +0100
From: "Yann E. MORIN" <yann.morin.1998@anciens.enib.fr>
To: uclibc@uclibc.org
Cc: Khem Raj <raj.khem@gmail.com>,
	Bernhard Reutner-Fischer <rep.dot.nop@gmail.com>,
	Carmelo AMOROSO <carmelo.amoroso@st.com>
Subject: [PATCH 4/7] ARM: transform the EABI/OABI choice into a boolean
Date: Sun,  9 Jan 2011 01:45:07 +0100
Message-Id: <1294533910-19305-5-git-send-email-yann.morin.1998@anciens.enib.fr>
X-Mailer: git-send-email 1.7.1
In-Reply-To: <1294533910-19305-1-git-send-email-yann.morin.1998@anciens.enib.fr>
References: <1294533910-19305-1-git-send-email-yann.morin.1998@anciens.enib.fr>

The CONFIG_ARM_OABI option is never used.

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@anciens.enib.fr>
Cc: Khem Raj <raj.khem@gmail.com>
Cc: Bernhard Reutner-Fischer <rep.dot.nop@gmail.com>
Cc: Carmelo AMOROSO <carmelo.amoroso@st.com>
---
 extra/Configs/Config.arm |   23 ++++++++---------------
 1 files changed, 8 insertions(+), 15 deletions(-)

Index: git/extra/Configs/Config.arm
===================================================================
--- git.orig/extra/Configs/Config.arm
+++ git/extra/Configs/Config.arm
@@ -12,23 +12,16 @@ config FORCE_OPTIONS_FOR_ARCH
 	default y
 	select ARCH_ANY_ENDIAN
 
-choice
-	prompt "Target ABI"
-	default CONFIG_ARM_EABI
+config CONFIG_ARM_EABI
+	bool "Build for EABI"
 	help
-	  If you choose "EABI" here, functions and constants required by the
-	  ARM EABI will be built into the library.  You should choose "EABI"
+	  If you say 'y' here, functions and constants required by the
+	  ARM EABI will be built into the library.  You should say 'y'
 	  if your compiler uses the ARM EABI, in which case you will also
-	  need a kernel supporting the EABI system call interface, or "OABI"
-	  for a compiler using the old Linux ABI.
-
-config CONFIG_ARM_OABI
-	bool "OABI"
-
-config CONFIG_ARM_EABI
-	bool "EABI"
+	  need a kernel supporting the EABI system call interface.
 
-endchoice
+	  If you say 'n' here, then the library will be built for the
+	  old Linux ABI.
 
 config COMPILE_IN_THUMB_MODE
 	bool "Build using Thumb mode"
